jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -204,6 +198,47 @@ private void uriMismatchWarningCheck(IType type, List<Diagnostic> diagnostics, I
         }
     }
 
+    /**
+     * Create an error diagnostic if a ServerEndpoint annotation does not follow the rules.
+     */
+    private void serverEndpointErrorCheck(IType type, List<Diagnostic> diagnostics, ICompilationUnit unit) throws JavaModelException {
+        for (IAnnotation annotation : type.getAnnotations()) {
+            if (annotation.getElementName().equals(WebSocketConstants.SERVER_ENDPOINT_ANNOTATION)) {
+                for (IMemberValuePair annotationMemberValuePair : annotation.getMemberValuePairs()) {
+                    if (annotationMemberValuePair.getMemberName().equals(WebSocketConstants.ANNOTATION_VALUE)) {
+                        String path = annotationMemberValuePair
+                                .getValue()
+                                .toString();
+                        Diagnostic diagnostic;
+                        if (!JDTUtils.hasLeadingSlash(path)) {
+                            diagnostic = createDiagnostic(annotation, unit,
+                                    WebSocketConstants.DIAGNOSTIC_SERVER_ENDPOINT_NO_SLASH,
+                                    WebSocketConstants.DIAGNOSTIC_SERVER_ENDPOINT);
+                            diagnostics.add(diagnostic);
```

Comment:
This diagnostic seems like it would be a good candidate for a quick fix, to prefix the string with a leading '/'. What do you think @giancarlopernudisegura? If you agree let's create a follow-up issue if we do not have one already

Reply:
I agree this would be a good quick fix. I will create a follow-up issue as there isn't one already.

Reply:
Created the issue: #284